assignment/core/src/main/java/com/assignment/core/config/RSA_AES_Config.java

```diff
@@ -0,0 +1,26 @@
+package com.assignment.core.config;
+
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "RSA and AES Configuration")
+public @interface RSA_AES_Config {
```

Comment:
The name for this config doesn't seem to be correct, is the purpose of this config only to store RSA and AES configs ?